### PR TITLE
trompeloeil: new recipe

### PR DESCRIPTION
**Pull request by svenihoney**
A thread-safe header-only mocking framework for C++11/14

### CODE REVIEW

File: meta-oe/recipes-test/trompeloeil/trompeloeil_v47.bb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+DESCRIPTION = "A popular statically typed mocking framework for C++14 and later"
+AUTHOR = "Björn Fahller, Andrew Paxie and others"
```
Comment (kraj) [inline meta-oe/recipes-test/trompeloeil/trompeloeil_v47.bb:2]: We have removed `AUTHOR` field see https://git.yoctoproject.org/poky/commit/?id=16dff2be01e8fc73e0057e3795319ea01fdeae5c

so please remove it.

**Review by kraj — CHANGES_REQUESTED** [review]

File: meta-oe/recipes-test/trompeloeil/trompeloeil_v47.bb (hunk 1 above)
Comment (svenihoney) [reply meta-oe/recipes-test/trompeloeil/trompeloeil_v47.bb]: Done. Thx for the hint!

Comment (kraj) [reply meta-oe/recipes-test/trompeloeil/trompeloeil_v47.bb]: please squash both commits into one and update the PR please

Comment (svenihoney) [reply meta-oe/recipes-test/trompeloeil/trompeloeil_v47.bb]: Check!

**Comment by kraj** [comment]
applied with f58552c014

**Pull request closed without merging**